test/reset-delay.test.js

```diff
@@ -7,16 +7,23 @@ import { internalFetch, slowDownAPI } from './helpers.js'
 import { convertToMs } from '../lib/helpers.js'
 import { DEFAULT_OPTIONS, HEADERS } from '../lib/constants.js'
 
-test('should reset the delay', async t => {
-  const fastify = Fastify()
-  const clock = FakeTimers.install()
+let fastify = null
+let clock = null
+
+const APP_RESPONSE_TEXT = 'Hello from the fastify-slow-down plugin'
+
+beforeEach(async t => {
+  fastify = Fastify()
+  clock = FakeTimers.install()
```

Comment:
these tests will fail unexpectedly because tap runs them in parallel. this was already a problem before, so let's avoid this pattern altogether. each test must instantiate its own fastify. if there is common set up to do (e.g. plugin registration and run configuration) we can consider factoring it into a function which we execute within each test, otherwise let's keep it simple and have each test instantiate fastify